stock-filters/ChirpVillage/PathGeneration.py

```diff
@@ -0,0 +1,85 @@
+from Mocks.Box import BoundingBox
+from Mocks.Level import Level
```

Comment:
These should be moved into the `if __name__ == "__main__":` block as the mocks should only be loaded when testing.